ynab_sdk/utils/configurations/cached.py

```diff
@@ -1,10 +1,27 @@
+from typing import Optional, Union
+
 from ynab_sdk.utils.configurations.default import DefaultConfig
 
 
 class CachedConfig(DefaultConfig):
 
-    def __init__(self, redis_host: str, redis_port: int, redis_db: int = 0, **kwargs):
+    def __init__(self, redis_host: str, redis_port: int, redis_db: int = 0, redis_pass: Optional[str] = None, **kwargs) -> None:
         super().__init__(**kwargs)
         self.redis_host = redis_host
         self.redis_port = redis_port
         self.redis_db = redis_db
+        self.redis_pass = redis_pass
+        self._redis_ttl: Optional[int] = 3600
+
+    @property
+    def redis_ttl(self) -> Union[int, None]:
```

Comment:
I missed this one: the return type is Optional[int]

Reply:
I missed it too...corrected